src/Raven.Studio/typescript/viewmodels/shell/registration.ts

```diff
@@ -147,6 +157,23 @@ class registration extends dialogViewModelBase {
         app.showBootstrapDialog(vm);
     }
 
+    forceLicenseUpdate() {
+        this.spinners.forceLicenseUpdate(true);
+        new forceLicenseUpdateCommand().execute()
+            .done(() => {
+                license.fetchLicenseStatus()
+                    .done(() => {
+                        const licenseStatus = license.licenseStatus();
+                        if (!licenseStatus.Expired &&
+                            !licenseStatus.ErrorMessage) {
+                            app.closeDialog(this);
+                        }
+                        license.fetchSupportCoverage();
+                    });
+            })
+            .always(() => this.spinners.forceLicenseUpdate(false));
```

Comment:
any failure/ success message here?

Reply:
will be shown by the forceLicenseUpdateCommand